src/test/java/htsjdk/samtools/cram/io/IOTestCases.java

```diff
@@ -72,6 +73,22 @@
         };
     }
 
+    @DataProvider(name = "testPositiveByteLists")
+    public static Object[][] testPositiveByteValues() {
+        final List<Byte> positiveByteTests = IOTestCases.byteTests()
+                .stream()
+                .filter(b -> b >= 0)
+                .collect(Collectors.toList());
+
+        final List<Byte> shuffled = new ArrayList<>(positiveByteTests);
+        Collections.shuffle(shuffled);
```

Comment:
Why shuffle these?  If you need to shuffle them can we use a set random seed?  `TestUtil.RANDOM_SEED` is a good option.

Reply:
results may be order-dependent, so I'm shuffling in order to give a good balance there.

I'll update to use the seed.